arangod/Aql/IndexHint.cpp

```diff
@@ -246,51 +243,48 @@ IndexHint::IndexHint(velocypack::Slice slice) {
     _waitForSync = basics::VelocyPackHelper::getBooleanValue(
         s, StaticStrings::WaitForSyncString, false);
 
-    type = basics::VelocyPackHelper::getStringValue(s, kFieldType, "");
+    type = basics::VelocyPackHelper::getStringValue(s, kFieldType, type);
```

Comment:
You can make `type` a `std::string_view` if you use ´ basics::VelocyPackHelper::getStringView` instead and pass `kTypeNone` as default argument. `kTypeNone` and the slice will remain valid for all uses of `type`.

Reply:
Done